### PR TITLE
You Played Yourself GIF command

### LINKED ISSUE #18: You Played Yourself GIF response (allison's idea, not mine)
**What would you like Steve to do?**
Steve should respond with the DJ Khaled you played yourself GIF using a phrase to trigger it. (probably along the lines of "steve, i played myself")

### PR DESCRIPTION
**Pull request by colinkautz**

### CODE REVIEW

File: src/commands/Fun/playedmyself.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export default class extends SteveCommand {
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			cooldown: 60,
+			cooldownLevel: 'channel',
+			aliases: ['i played myself'],
+			description: 'congratulations, you played yourself.',
+			extendedHelp: 'this command returns the DJ Khaled GIF'
```
Comment (jwford) [inline src/commands/Fun/playedmyself.ts:13]: few changes needed here: command options should be alphabetized, description + extendedHelp should be capitalized properly, and the alias shouldn't have any spaces

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "steve",
-  "version": "3.3.0",
+  "version": "3.4.0",
```
Comment (jwford) [inline package.json]: Can you change the version numbers here + in the lockfile back to 3.3.0? This command will be included as part of 3.4.0 but there will probably be more changes before the version is incremented + released.

**Review by jwford — CHANGES_REQUESTED** [review]
Looks good, left comments in the places where things need to be changed slightly

File: package.json (hunk 2 above)
Comment (colinkautz) [reply package.json]: changed back to 3.3.0

File: src/commands/Fun/playedmyself.ts (hunk 1 above)
Comment (jwford) [inline src/commands/Fun/playedmyself.ts]:
```suggestion
			aliases: ['iplayedmyself'],
			cooldown: 60,
			cooldownLevel: 'channel',
			description: 'Congratulations, you played yourself.',
			extendedHelp: 'This command returns the DJ Khaled GIF. You know the one.'
```

**Review by jwford — DISMISSED** [review]

**Review by BoedJ — APPROVED** [review]

File: src/commands/Fun/playedmyself.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+		super(store, file, directory, {
+			aliases: ['iplayedmyself'],
+			cooldown: 60,
+			cooldownLevel: 'channel',
+			description: 'Congratulations, you played yourself.',
+			extendedHelp: 'This command returns the DJ Khaled GIF. You know, that DJ Khaled GIF.'
+		});
+	}
+
+	async run(msg: KlasaMessage): Promise<Message> {
```
Comment (BoedJ) [inline src/commands/Fun/playedmyself.ts]:
```suggestion
	public async run(msg: KlasaMessage): Promise<Message> {
```

**Review by BoedJ — CHANGES_REQUESTED** [review]
Could you just add public to the run function as proposed?

**Review by jwford — DISMISSED** [review]

**Comment by jwford** [comment]
I checked out this PR locally and Discord won't load the GIF, and when I try to view it in the Photos app, this is what appears. So...maybe put in a new verison of the file lol 
image

**Review by BoedJ — DISMISSED** [review]

**Comment by jwford** [comment]
i'll give this a whirl when i have a free moment or two

File: src/commands/Fun/playedmyself.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+export default class extends SteveCommand {
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			aliases: ['iplayedmyself'],
+			cooldown: 60,
+			cooldownLevel: 'channel',
+			description: 'Congratulations, you played yourself.',
+			extendedHelp: 'This command returns the DJ Khaled GIF. You know, that DJ Khaled GIF.'
```
Comment (jwford) [inline src/commands/Fun/playedmyself.ts]: internationalizationnnnnnnn

File: src/commands/Fun/playedmyself.ts [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+		});
+	}
+
+	public async run(msg: KlasaMessage): Promise<Message> {
+		return msg.channel.send({ files: [{ attachment: './assets/images/played_yourself.mp4', name: 'played_yourself.mp4' }] });
```
Comment (jwford) [inline src/commands/Fun/playedmyself.ts]: image description pls pls

**Review by jwford — CHANGES_REQUESTED** [review]

File: src/commands/Fun/playedmyself.ts (hunk 4 above)
Comment (colinkautz) [reply src/commands/Fun/playedmyself.ts]: added i18n in this recent push

File: src/commands/Fun/playedmyself.ts (hunk 5 above)
Comment (colinkautz) [reply src/commands/Fun/playedmyself.ts]: added image description in this recent push

**Pull request closed without merging**